__tests__/shapes.test.js

```diff
@@ -35,15 +35,32 @@ describe('Shape Module', () => {
                             'Auto-fit test - width and height of shape object should not be specified. This shape should auto-grow to fit this text. ',
                         autoFit: true,
                     });
+                    // TODO: FIXME: auto-shrink text broke (well, it didn't work perfectly before either - when opening a PPTX with an auto-shrink text shape, the text
```

Comment:
Yup - that's still an issue unfortunately. But we never had a real-world case of needing to use auto-shrinking text. However, this might come up in the DecisionLab additions to the RACI matrix.